pytest/lib/cluster.py

```diff
@@ -156,15 +157,21 @@ def get_changes(self, changes_request):
     def validators(self):
         return set(map(lambda v: v['account_id'], self.get_status()['validators']))
 
+    def stop_checking_refmap(self):
+        self.is_check_refmap = False
```

Comment:
Where is this used?

Reply:
In any adversarial tests that affect the way of chain construction.